sql/step_91_add_dicom_data_table.sql

```diff
@@ -0,0 +1,30 @@
+-- add foreign key from dicom to dicom_data
+-- this means that the dicom table will be responsible in maintaining the relationship between dicom and the dicom data table
+ALTER TABLE DICOM ADD COLUMN DICOM_DATA_ID int(11);
+
+-- add the new table
+-- note that:
+-- 1. adding primary key with auto increment. We will not be setting this auto increment manually
+-- 2. adding TEMP_DATA_ID for migration purpose, so that we can relate existing data from DICOM table to this new table while doing migration
+CREATE TABLE DICOM_DATA (
+                                       DICOM_DATA_ID int NOT NULL AUTO_INCREMENT,
```

Comment:
all columns in a table are prefixed by some abbreviation of the table name , some something like `DD.DICOM_DATA_IT`

Reply:
Added prefix DMD_